spec/jobs/delete_missed_reservations_job_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe DeleteMissedReservationsJob, type: :job do
+  it 'deletes appropriate reservations' do
+    mock_app_config(res_exp_time: 5)
+    FactoryGirl.create(:missed_reservation,
+                       start_date: Time.zone.today - 11.days,
+                       due_date: Time.zone.today - 10.days)
+    missed = FactoryGirl.create(:missed_reservation,
+                                start_date: Time.zone.today - 4.days,
+                                due_date: Time.zone.today - 3.days)
+    valid = FactoryGirl.create(:valid_reservation)
+    described_class.perform_now
+    expect(Reservation.all).to match_array([missed, valid])
```

Comment:
This is a test that was carried over from when the query to get deletable missed reservations was defined in the job, so I wanted to make sure that the query/general behavior of the job was working properly. I don't think that it's necessary anymore, so I'll get rid of it